src/components/formControls/select/style.ts

```diff
@@ -122,3 +122,45 @@ export const StyledOptionText = styled<StyleProps, 'div'>('div')`
   white-space: nowrap;
   text-overflow: ${p => p.showAllContents ? 'initial' : 'ellipsis'};
 `
+
+export const StyledOptionsOverlay = styled<StyleProps, 'div'>('div')`
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(12,13,33,0.85);
+  align-items: center;
+  z-index: 999;
+  justify-content: center;
+`
+
+export const StyledOptionsModal = styled<StyleProps, 'div'>('div')`
+  border-radius: 7px;
+  background: #fff;
+  height: 332px;
```

Comment:
The fixed height here is a bit peculiar to me. Is this a design decision? On larger screens it seems a lot of space is wasted and the modal uneccessarily scrolls:
![image](https://user-images.githubusercontent.com/741836/46562160-b073a100-c8af-11e8-91aa-660aa185a6d2.png)

Reply:
I had the design in abstract to go off of, which just showed the modal concept on a smaller smartphone viewport, so I lifted the values from that.

We should probably do a percentage for width/height and aim for the modal to take up a certain portion of the window. wdyt? cc: @jenn-rhim

Reply:
@petemill 
The height is now a variable percentage to accommodate different viewports, iPad example:

<img width="619" alt="screen shot 2018-10-09 at 9 13 09 am" src="https://user-images.githubusercontent.com/8732757/46683179-671f9c00-cba4-11e8-96e0-1321e513f936.png">